### DISCUSSION TITLE
v1.0

### DISCUSSION THREAD

**Discussion by dala318** [dala318/nordpool_planner#21] [Announcements]
## What's Changed
* Create FUNDING.yml by @dala318 in https://github.com/dala318/nordpool_planner/pull/19
* Update FUNDING.yml by @dala318 in https://github.com/dala318/nordpool_planner/pull/20

**Full Changelog**: https://github.com/dala318/nordpool_planner/compare/0.2.4...1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/dala318/nordpool_planner/releases/tag/1.0.0'>v1.0.0</a>.</em>

**Comment by CommanderROR9** [comment]
Sadly the "old" bug of the Sensor not actually turning off after 2 hours when set to 2 hours is still there...

**Reply by dala318** [reply]
Could very well be, have had limited time to work in this integration last year or so. (help appreciated and accept external pull-requests if anyone feel for it)
Which Issue # are you referring to?

But please keep in mind that the integration does not look in the past. So if you are in a "smaller" search_length and prices continuously increase the cheapest price over the "duration" can be "now" for an extended period of time.
